bfapi/service/jobs.py

```diff
@@ -219,6 +208,26 @@ def create(
         tide_max_24h=tide_max,
     )
 
+def __genCmd(
```

Comment:
renamed to `__make_cli_cmd`.  The `__` prefix is deliberate.  There is no reason for this function to ever be called externally, and scoping is valuable.